website/docs/sys-mgmt/events/notification_rule_expression_policies.mdx

```diff
@@ -6,9 +6,36 @@ title: Notification Rule Expression Policies
 
 Notification rules with bound expression policies are very powerful. The following are examples of what can be achieved.
 
+### Trigger alert when user logs in from unknown device
+
+This expression is triggered when a user logs in from an unknown device. This can be combined with a notification rule to alert administrators, or the user in question if `Send notification to event user` is enabled on the notification rule.
```

Comment:
```suggestion
This expression policy is triggered when a user logs in from an unknown device. This can be combined with a notification rule to alert administrators, or the user in question if `Send notification to event user` is enabled on the notification rule.
```